hack/test-cmd.sh

```diff
@@ -460,8 +464,8 @@ osadm policy add-role-to-user admin adduser -n ui-test-project
 # Make sure project can be listed by osc (after auth cache syncs)
 sleep 2 && [ "$(osc get projects | grep 'ui-test-project')" ]
 # Make sure users got added
-[ "$(osc describe policybinding master:default -n ui-test-project | grep createuser)" ]
```

Comment:
is the leading colon needed? wouldn't something without a namespace component be a cluster binding?